src/Admin/AbstractAdmin.php

```diff
@@ -1469,7 +1491,7 @@ public function getSideMenu($action, ?AdminInterface $childAdmin = null)
             return $this->getParent()->getSideMenu($action, $this);
         }
 
-        $this->buildSideMenu($action, $childAdmin);
```

Comment:
I guess this is BC break for someone overriding `buildSideMenu`.

Reply:
Changed